script/Constants.s.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.25;
+
+// Factories
+address constant GNOSIS_CURVE_STABLE_SWAP_FACTORY = 0xbC0797015fcFc47d9C1856639CaE50D0e69FbEE8;
+
+// Liquidity Pools
+address constant GNOSIS_CURVE_POOL_XDAI_BREAD = 0xf3D8F3dE71657D342db60dd714c8a2aE37Eac6B4;
+
+// Tokens
+address constant GNOSIS_BREAD = 0xa555d5344f6FB6c65da19e403Cb4c1eC4a1a5Ee3;
+address constant XDAI = 0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d;
```

Comment:
Probably also add `GNOSIS_` prefix here too. Fwiw, my original comment was mostly just so that it was obvious that these contract values are the ones deployed to Gnosis "mainnet" and not a testnet or local address. `GNOSIS_` prefix works for me.